cli/tests/e2e/test_api.py

```diff
@@ -8,7 +8,7 @@
 
 
 @pytest.mark.slow
-def test_api(capsys, run_semgrep_in_tmp):
+def test_api(unique_home_dir, capsys, run_semgrep_in_tmp):
```

Comment:
Can you explain (in comments in the code) why we need unique_home_dir here? I've read the changelog but I still don't understand
why we got those flaky errors.

Reply:
Added!